src/PHPCR/Util/QOM/Sql2Scanner.php

```diff
@@ -152,7 +152,7 @@ protected function scan($sql2)
         $splitString = \str_split($sql2);
         for ($index = 0; $index < count($splitString); $index++) {
             $character = $splitString[$index];
-            if (!$stringStartCharacter && in_array($character, [' ', "\t", "\n"], true)) {
+            if (!$stringStartCharacter && in_array($character, [' ', "\t", "\n", "\r"], true)) {
```

Comment:
if you do not have this line, do the tests actually fail? do we have a test that builds a query with `PHP_EOL`? if that is the case, lets have a windows build.

if not, i suggest that we craft queries with explicit `\n` and `\r` newlines to cover all cases (macos, linux and windows) instead of building on a windows system.

Reply:
oh, found the reproducer commit and build you linked. so it does only fail on windows. but why is that? shouldn't that test also fail on linux?

Reply:
@dbu I think git converts line breaks on windows from "\n" to the windows "\n\r" format. And that's why its also failing there.

Reply:
so if we would add a test that has a `\n\r` in it, it would also fail on linux (without the fix)? 

i think we should have such a test as well then, it seems a bit indirect and obscure that the windows test happens the way it does.